core/io/resource_format_binary.cpp

```diff
@@ -85,15 +85,18 @@ enum {
 	VARIANT_VECTOR4 = 50,
 	VARIANT_VECTOR4I = 51,
 	VARIANT_PROJECTION = 52,
+	VARIANT_PACKED_VECTOR4_ARRAY = 53, // For compat with 4.3, parsing only.
 	OBJECT_EMPTY = 0,
 	OBJECT_EXTERNAL_RESOURCE = 1,
 	OBJECT_INTERNAL_RESOURCE = 2,
 	OBJECT_EXTERNAL_RESOURCE_INDEX = 3,
-	// Version 2: added 64 bits support for float and int.
-	// Version 3: changed nodepath encoding.
-	// Version 4: new string ID for ext/subresources, breaks forward compat.
+	// Version 2: Added 64 bits support for float and int.
+	// Version 3: Changed nodepath encoding.
+	// Version 4: New string ID for ext/subresources, breaks forward compat.
 	// Version 5: Ability to store script class in the header.
 	FORMAT_VERSION = 5,
+	// Version 6: Added PackedVector4Array variant. Parsing only.
+	FORMAT_VERSION_READABLE = 6,
```

Comment:
I notice that I forgot to add code to make use of this... but it's also pointless, as resources saved in 4.3 will have the `4.3` Godot version and we don't support reading binary formats from newer Godot versions. So this part of the PR can be reverted.